### PR TITLE
IH-39 Add api get all enabled posts

### PR DESCRIPTION
**Pull request by Sotatek-KimMai**

### CODE REVIEW

File: src/main/java/art/heredium/domain/post/model/dto/response/GetEnabledPostsResponse.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package art.heredium.domain.post.model.dto.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class GetEnabledPostsResponse {
```
Comment (Sotatek-ThoLe) [inline src/main/java/art/heredium/domain/post/model/dto/response/GetEnabledPostsResponse.java:10]:
```suggestion
public class PostResponse{
```

File: src/main/java/art/heredium/controller/user/UserPostController.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import art.heredium.service.PostService;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/user/posts")
+public class UserPostController {
+
+  private final PostService postService;
+
+  @GetMapping("/enabled")
```
Comment (Sotatek-ThoLe) [inline src/main/java/art/heredium/controller/user/UserPostController.java:22]:
```suggestion
  @GetMapping("/enabled-list")
```

File: src/main/java/art/heredium/domain/post/model/dto/response/GetEnabledPostsResponse.java (hunk 1 above)
Comment (Sotatek-KimMai) [reply src/main/java/art/heredium/domain/post/model/dto/response/GetEnabledPostsResponse.java]: I updated in https://github.com/Incuvers-IT/heredium-BE/pull/20/commits/91ecc1dd61f82ec063b9be213e5748ef93cc06b2

File: src/main/java/art/heredium/controller/user/UserPostController.java (hunk 2 above)
Comment (Sotatek-KimMai) [reply src/main/java/art/heredium/controller/user/UserPostController.java]: I updated in https://github.com/Incuvers-IT/heredium-BE/pull/20/commits/91ecc1dd61f82ec063b9be213e5748ef93cc06b2

File: src/main/java/art/heredium/domain/post/model/dto/response/PostResponse.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+@Getter
+@Setter
+@AllArgsConstructor
+public class PostResponse {
+  private Long id;
+  private String name;
+  private String imageUrl;
+  private Boolean isEnabled;
+  private String contentDetail;
+  private String navigationLink;
```
Comment (Sotatek-ThoLe) [inline src/main/java/art/heredium/domain/post/model/dto/response/PostResponse.java:16]: Thêm @JsonProperty để map sang snake_case

Comment (Sotatek-KimMai) [reply src/main/java/art/heredium/domain/post/model/dto/response/PostResponse.java]: I updated in https://github.com/Incuvers-IT/heredium-BE/pull/20/commits/e45e93585bafd22ee3454c83a2eaa376d24135b0

**Review by Sotatek-ThoLe — APPROVED** [review]

**Pull request merged**